backend/src/youtube-feed/youtube-feed.service.ts

```diff
@@ -0,0 +1,69 @@
+
+import { ConfigService } from '../config/config.service';
+import { Injectable, Logger } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { YouTubeFeed } from './youtube-feed.entity';
+import { TransactionRepository, Repository, Transaction } from 'typeorm';
+import axios from 'axios';
+
+const config = new ConfigService();
+
+const API_KEY = config.youTubeApiKey;
+const CHANNEL_ID = config.youTubeChannelID;
+const MAX_RESULTS = config.youTubeChannelListLimit;
+
+const youtube_fetch_url = `https://www.googleapis.com/youtube/v3/search?`
+    + `key=${API_KEY}`
```

Comment:
you are already using tagged template expression `the \`` It support multiline.
```
`You can write
 code like this
 and ${inject} thing just fine
 `
```

Reply:
yes thank you